src/Bundle/DataSourceBundle/DependencyInjection/FSIDataSourceExtension.php

```diff
@@ -69,6 +72,9 @@ private function registerDrivers(LoaderInterface $loader): void
         $loader->load('driver/collection.xml');
         $loader->load('driver/doctrine-orm.xml');
         $loader->load('driver/doctrine-dbal.xml');
+        if (true === class_exists(Client::class)) {
```

Comment:
Should we also not check for existance of elastica extension?

Reply:
`ElasticaDriver` does not require `FOSElasticaBundle` to be loaded if that's what you mean. `FOSElasticaBundle` has to be loaded to use its `ElasticaToModelTransformerInterface` implementing services. That setup is verified and loaded in `FOSElasticaPass`